### ISSUE TITLE
Add NodeJS to the PHP image

### ISSUE THREAD

**Issue by LucasCalazans** [magento/magento-cloud-docker#32]
### Summary

Add the latest version of `node`, `npm` and `grunt-cli`

### Expected result
1. As a frontend developer, I should have to have the ability to use `grunt-cli` inside a PHP container (like the `fpm`)
2. I need `node`/`npm`/`grunt-cli` inside a `php` container because some tasks use PHP and database connection

### Actual result
1. We don't have the ability to use `node`, `npm` or `grunt-cli`

### Suggestion

Add a new `RUN` command to Dockerfile, something like this:
```yaml
RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - \
    && apt-get install -y --no-install-recommends nodejs \
    && npm install -g grunt-cli

```

**Comment by YPyltiai** [comment]
@shiftedreality , @bbatsche , please review

**Comment by sorinsugar** [comment]
Wen you build the docker environment using the ece-tools command, you can link relating services, one of these services is node:
`/vendor/bin/ece-tools docker:build --node [6|8|10|11]`

This container will give you the ability to run `node` or `npm` by default etc.

**Comment by LucasCalazans** [comment]
@sorinsugar yeah, it is true, but some grunt's tasks needs PHP because it will run a static deploy or something like that. So we need node inside a PHP container.

You can take a look at this file:
https://github.com/magento/magento2/blob/2.3-develop/dev/tools/grunt/configs/combo.js#L19

**Comment by shiftedreality** [comment]
Hello @LucasCalazans,

Thank you for submitting this request. I've created task MAGECLOUD-3953 to add this functionality.

**Comment by shiftedreality** [comment]
Related PR: https://github.com/magento/magento-cloud-docker/pull/34

**Comment by LucasCalazans** [comment]
Fixed on #34
